bin/pakiti-client

```diff
@@ -238,6 +238,25 @@ sub find_packages ($) {
     my($data) = @_;
     my($cmd, $format, @list);
     my(@output);
+	
+    #The user wants to send svmon data
+    if ($Option{"svmonreport"}) {
+        $cmd = $Option{"svmon"};
```

Comment:
You probably want to make the user's life easier and check that the $cmd is non-empty here (and possibly trigger an error). Otherwise the code fails later with a potentially cryptic message (when -s is specified and svmon isn't in the PATH).